config/database.yml

```diff
@@ -10,8 +10,8 @@ default: &default
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   host: localhost
   timeout: 5000
-  username: recipe_app
-  password: <%= ENV['RECIPE_APP_DATABASE_PASSWORD'] %>
+  username: postgres
```

Comment:
[Optional] To avoid conflicts in the future, I think we should extract the database username and password into environment variables.

Reply:
I will follow your recommendations